apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -104,6 +104,28 @@ function get_theme_slug() {
 	return $theme_slug;
 }
 
+/**
+ * Returns a normalized slug for the current theme.
+ *
+ * In some cases, the theme is located in a subfolder like `pub/maywood`. Use
+ * this function to get the slug without the prefix.
+ *
+ * @param string $theme_slug The raw theme_slug to normalize.
+ * @return string Theme slug.
+ */
+function normalize_theme_slug( $theme_slug ) {
```

Comment:
copied straight from here: https://github.com/Automattic/wp-calypso/pull/37434/files#diff-ac595e0aacfc361a74e65bf750e8e43bR119

Reply:
I'm confident you'll remove this too in that other PR 😄